<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="GitHub Suite" duration-ms="23701" started-at="2019-11-03T21:54:01Z" finished-at="2019-11-03T21:54:25Z">
    <groups>
    </groups>
    <test name="Automation Positive Tests" duration-ms="23701" started-at="2019-11-03T21:54:01Z" finished-at="2019-11-03T21:54:25Z">
      <class name="test.GitHubTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:test.GitHubTest@5afa04c]" name="setup" is-config="true" duration-ms="21407" started-at="2019-11-03T21:54:01Z" finished-at="2019-11-03T21:54:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="getResponse()[pri:0, instance:test.GitHubTest@5afa04c]" name="getResponse" duration-ms="1926" started-at="2019-11-03T21:54:23Z" finished-at="2019-11-03T21:54:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResponse -->
        <test-method status="PASS" signature="showFirstResultName_Star()[pri:0, instance:test.GitHubTest@5afa04c]" name="showFirstResultName_Star" duration-ms="1" started-at="2019-11-03T21:54:25Z" depends-on-methods="test.GitHubTest.getResponse" finished-at="2019-11-03T21:54:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- showFirstResultName_Star -->
        <test-method status="PASS" signature="showLastReleaseTagResult()[pri:0, instance:test.GitHubTest@5afa04c]" name="showLastReleaseTagResult" duration-ms="320" started-at="2019-11-03T21:54:25Z" depends-on-methods="test.GitHubTest.showFirstResultName_Star" finished-at="2019-11-03T21:54:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- showLastReleaseTagResult -->
        <test-method status="FAIL" signature="verifyLastedReleaset()[pri:0, instance:test.GitHubTest@5afa04c]" name="verifyLastedReleaset" duration-ms="14" started-at="2019-11-03T21:54:25Z" depends-on-methods="test.GitHubTest.showLastReleaseTagResult" finished-at="2019-11-03T21:54:25Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at test.GitHubTest.verifyLastedReleaset(GitHubTest.java:101)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLastedReleaset -->
      </class> <!-- test.GitHubTest -->
    </test> <!-- Automation Positive Tests -->
  </suite> <!-- GitHub Suite -->
</testng-results>
